first_task.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+from pyspark.sql import SparkSession
+from data import info_cinema, info_ratings
+from pyspark.sql.functions import col
```

Comment:
Imports should be grouped in the following order:

- Standard library imports.
- Related third party imports.
- Local application/library specific imports.

You should put a blank line between each group of imports.

Reply:
okey